images/build/php/8.3/Dockerfile

```diff
@@ -0,0 +1,3 @@
+# Build PHP 8.3
+FROM oryxdevmcr.azurecr.io/private/oryx/php-build-prereqs AS php83-build
```

Comment:
do we have an understanding of why we have these separate python and php build image dockerfiles?

Reply:
let me come back on this, saw the usage some where in the code.